### PR TITLE
Modified content to adhere better to IBM style guide

### PR DESCRIPTION
**Pull request by jc-berger**
Fixes #254 Review Language in Authoring Stacks.

Since I am doing a review of all the authoring stacks docs, I'll be making several PRs. I don't want any single PR to be too large, so I'll make a new PR for every several docs that I modify to meet the IBM Style Guide's requirements.

### CODE REVIEW

**Review by maysunfaisal — APPROVED** [review]
/lgtm

re-adding lgtm on new PR since there were issues rebasing with the prev PR

**Comment by jc-berger** [comment]
@rkratky ready to merge if you think it looks good, thanks!

File: docs/modules/user-guide/partials/proc_adding-a-name-to-a-devfile.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 = Adding a name to a devfile
 
 [role="_abstract"]
-Adding a name to a devfile is mandatory. Both `name` and `generateName` are optional attributes, but at least one of them must be defined.
+Adding a name to a devfile is mandatory. Both `name` and `generateName` are optional attributes, but you must define at least one of them..
```
Comment (rkratky) [inline docs/modules/user-guide/partials/proc_adding-a-name-to-a-devfile.adoc:5]:
```suggestion
Adding a name to a devfile is mandatory. Use the `name` or `generateName` attributes (or both) to define the name.
```

**Review by rkratky — APPROVED** [review]
Just one nit.

**Comment by jc-berger** [comment]
@rkratky , thanks! Accepted the change, so it's ready to merger :)

**Comment by rkratky** [comment]
For more work on these Eclipse Che related content concerns, please see issue #491: devfile/api#491

**Pull request merged**